2022/70-oriol-arviz.md

```diff
@@ -0,0 +1,42 @@
+# Oriol Abril Pla: Contributing to ArviZ and Open Source: Social and Technical Sides
+
+## Upcoming Events
+Join our Meetup group for more events!
+https://www.meetup.com/data-umbrella
+
+## Key Links
+- Transcript: https://github.com/data-umbrella/event-transcripts/blob/main/2022/ 
```

Comment:
@Cristinamulas can you add in the file name here?

Reply:
@Cristinamulas done!